octavia-cli/octavia_cli/init/commands.py

```diff
@@ -3,12 +3,39 @@
 #
 
 import os
-from typing import Iterable, Tuple
+from typing import Iterable, Tuple, Dict, Optional
 
 import click
+import yaml
+
 from octavia_cli.base_commands import OctaviaCommand
 
 DIRECTORIES_TO_CREATE = {"connections", "destinations", "sources"}
+API_HEADERS_CONFIGURATION_FILE = "api_headers_configuration.yaml"
+DEFAULT_HEADERS = {
+    "headers": [
+        {"name": "Content-Type", "value": "application/json"},
+    ]
+}
+OCTAVIA_ENV_FILE_NAME = ".octavia"
+AIRBYTE_HEADERS_FILE_PATH_ENV_VARIABLE_NAME = "AIRBYTE_HEADERS_FILE_PATH"
+
+
+def create_api_headers_configuration_file(file_name: str, example_headers: Dict[str, str]) -> bool:
+    if not os.path.isfile(file_name):
+        with open(file_name, "w") as file:
+            yaml.dump(example_headers, file)
+        return True
+    return False
+
+
+def create_octavia_env_file(file_name: str, env_variables: Dict[str, str]) -> bool:
+    file_content = "\n".join([f"{key}={value}" for key, value in env_variables.items()])
+    if not os.path.isfile(file_name):
+        with open(file_name, "w") as file:
+            file.write(file_content)
+        return True
+    return False
 
 
 def create_directories(directories_to_create: Iterable[str]) -> Tuple[Iterable[str], Iterable[str]]:
```

Comment:
Please run `create_api_headers_configuration_file` outside of the `create_directories` function, directly for `def init`

Reply:
I dont know if I understand that correctly.

Reply:
I mean that you are creating the API headers file in a function whose role is to create directories. For separation of concerns I think you can call `create_api_headers_configuration_file` in this `init` function, after calling `create_directories`.